test/Libraries/DynamoMSOfficeTests/DynamoMSOfficeTests.csproj

```diff
@@ -11,16 +11,24 @@
     <RootNamespace>DynamoMSOfficeTests</RootNamespace>
     <AssemblyName>DynamoMSOfficeTests</AssemblyName>
   </PropertyGroup>
-  <ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net48' ">
 	  <PackageReference Include="NUnit" Version="2.6.3" />
     <PackageReference Include="NUnitTestAdapter" Version="2.3.0" ExcludeAssets="all" />
       <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.3.2" />
     <Reference Include="PresentationCore" />
     <Reference Include="System" />
     <Reference Include="System.Core" />
```

Comment:
These are common to both net48 and net60 so can be listed outside of the two item groups.

Reply:
Solved